charts/plex/templates/deployment.yaml

```diff
@@ -181,16 +195,21 @@ spec:
             httpGet:
               path: /identity
               port: 32400
-            initialDelaySeconds: {{ .Values.probes.readiness.initialDelaySeconds }}
             failureThreshold: {{ .Values.probes.readiness.failureThreshold }}
-            timeoutSeconds: {{ .Values.probes.readiness.timeoutSeconds }}
+            periodSeconds: {{ .Values.probes.readiness.periodSeconds }}
           livenessProbe:
             httpGet:
               path: /identity
               port: 32400
-            initialDelaySeconds: {{ .Values.probes.liveness.initialDelaySeconds }}
             failureThreshold: {{ .Values.probes.liveness.failureThreshold }}
-            timeoutSeconds: {{ .Values.probes.liveness.timeoutSeconds }}
+            periodSeconds: {{ .Values.probes.liveness.periodSeconds }}
+          startupProbe:
```

Comment:
`StartupProbe` wasn't introduced to kubernetes until 1.16, we may cause failures for folks running < 1.16 clusters with this change.  Do you think we should wrap this one in a version check or possibly an 'enabled' check?

Reply:
@billimek this might bring up a bigger issue. Like do we plan to have our charts support older versions of k8s? Since we're running these at home I would expect people to be at least 1.17 or higher (or current version minus 2)

Reply:
It is indeed a bigger question.  I, personally, feel that we shouldn't need to worry about k8s clusters older than 1.16 for charts in this repo but am trying to be conscious of folks who (for whatever reason) are running older versions.  Maybe that's not really a concern in a k8s-at-home type environment, like you say.

Why don't we go forward with the idea that these charts are intended for clusters at current version minus 2 and above?  If it becomes a problem, we can revisit.  I'd rather not have us clog-up charts with a bunch of conditionals (checking k8s versions, etc) unless it is really necessary.

For reference, here is the kubernetes release version and date chart:

![image](https://user-images.githubusercontent.com/6393612/91563915-b80e1a00-e90d-11ea-803f-21f50460d795.png)

Reply:
Agreed, we will need to make a note of this in the upcoming README for the chart repo.